intellij/build.gradle.kts

```diff
@@ -14,30 +12,52 @@ val ideProfile = IdeVersions.ideProfile(project)
 val publishToken: String by project
 val publishChannel: String by project
 
+val resharperDlls = configurations.create("resharperDlls") {
+    isCanBeConsumed = false
```

Comment:
What does this do?

Reply:
other projects can't use consume it, this project is a producer of the configuration

Reply:
https://docs.gradle.org/current/userguide/declaring_dependencies.html describes this, see table 1